aiobotocore/endpoint.py

```diff
@@ -13,15 +16,23 @@
     logger,
 )
 from botocore.hooks import first_non_none_response
+from requests.models import Response
 
 from aiobotocore.httpchecksum import handle_checksum_body
 from aiobotocore.httpsession import AIOHTTPSession
 from aiobotocore.response import StreamingBody
 
+try:
+    import httpx
+except ImportError:
+    httpx = None
+
 DEFAULT_HTTP_SESSION_CLS = AIOHTTPSession
 
 
-async def convert_to_response_dict(http_response, operation_model):
```

Comment:
While I personally like type annotations a lot, this code is based on upstream (botocore) and we strive to keep it closely aligned to simplify version bumps. Please do not add type annotations as part of this PR. Thanks!

Reply:
ya fact of life :(